ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml

```diff
@@ -135,27 +135,51 @@ presubmits:
       serviceAccountName: ci-operator
     trigger: (?m)^/test( | .* )e2e-create-20190930,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
+    branches:
+    - master
+    context: e2e-create-20191027-public
+    decorate: true
+    name: pull-ci-azure-master-e2e-create-20191027-public
+    rerun_command: /test e2e-create-20191027-public
+    spec:
+      containers:
+      - args:
+        - hack/tests/e2e-create.sh
+        env:
+        - name: API_VERSION
+          value: 2019-10-27-preview
+        - name: FOCUS
+          value: \[EveryPR\]
+        image: registry.svc.ci.openshift.org/azure/ci-base:latest
+        name: ""
+        resources: {}
+      serviceAccountName: ci-operator
+    trigger: (?m)^/test( | .* )e2e-create-20191027-public,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
     branches:
     - master
-    context: e2e-create-20191027
+    context: e2e-create-20191027-private
     decorate: true
-    name: pull-ci-azure-master-e2e-create-20191027
-    rerun_command: /test e2e-create-20191027
+    name: pull-ci-azure-master-e2e-create-20191027-private
+    rerun_command: /test e2e-create-20191027-private
     spec:
       containers:
       - args:
         - hack/tests/e2e-create.sh
         env:
         - name: API_VERSION
           value: 2019-10-27-preview
+        - name: PRIVATE_API_SERVER
+          value: "true"
```

Comment:
is the proxy server set anywhere? PROXYURL_$REGION

Reply:
it should come from the secrets. I set it in `secrets/secrets`